tool/mustachio/codegen_runtime_renderer.dart

```diff
@@ -346,8 +346,10 @@ class ${renderer._rendererClassName}${renderer._typeParametersString}
     var contextClass = renderer._contextClass;
     var generics = renderer._typeParametersStringWith(
         '$_contextTypeVariable extends ${renderer._typeName}');
-    _buffer.writeln('static Map<String, Property<$_contextTypeVariable>> '
-        'propertyMap$generics() => {');
+    _buffer.writeln('''
+    static final Map<Type, Object> _propertyMapCache = {};
```

Comment:
Usually I object to process-global caching like this, but in this case, I think it is probably fine as this is literally caching a structure derived from the structure of the program, which... at least, unless there is something very very wrong in my understanding... is unchanging at the Isolate level.   The only reason to not have it then would be if mustachio code is ever used to process multiple wildly different templates and you didn't want to hold the memory in between.

Might be worth a comment?

Reply:
Yes, that's correct. I could conceivably even just hardcode a property map for each generic type which is actually in play, but this I think might get complicated...

And yeah the data never changes; honestly it could probably be an LRU cache of size ~1 or ~2, and I'd be open to using quiver's LRU map, but I think we recently ripped out quiver...

Reply:
Given that, a comment is probably worth something here, but otherwise this is OK.

Reply:
I've added a comment with the info I think you're looking for.